modules/swift/examples/code_snippets/SampleCodeTest.swift

```diff
@@ -1209,2824 +1208,1603 @@ class SampleCodeTest {
+        
+        // end::p2p-act-rep-config-cacert[]
+        // tag::p2p-act-rep-config-self-cert[]
+        // Configure Server Security -- only accept self-signed certs
+        config.acceptOnlySelfSignedServerCertificate = true // <.>
+        
+        // end::p2p-act-rep-config-self-cert[]
+        // tag::p2p-act-rep-config-pinnedcert[]
+        // Return the remote pinned cert (the listener's cert)
+        config.pinnedServerCertificate = cert // Get listener cert if pinned
+        
+        // end::p2p-act-rep-config-pinnedcert[]
+        // Configure Client Security // <.>
+        // tag::p2p-act-rep-auth[]
+        //  Set Authentication Mode
+        config.authenticator = BasicAuthenticator(username: validUsername, password: validPassword)
+        
+        // end::p2p-act-rep-auth[]
+        // end::p2p-act-rep-config-tls-full[]
+        
+        // tag::p2p-tlsid-tlsidentity-with-label[]
+        // Check if Id exists in keychain and if so, use that Id
+        if let tlsIdentity = try TLSIdentity.identity(withLabel: "doco-sync-server") { // <.>
+            print("An identity with label : doco-sync-server already exists in keychain")
+            config.authenticator = ClientCertificateAuthenticator(identity: tlsIdentity) // <.>
+        }
+        
+        // end::p2p-tlsid-check-keychain[]
+        // end::p2p-tlsid-tlsidentity-with-label[]
+    }
+    
+    // tag::p2p-tlsid-manage-func[]
+    func myGetCert() throws -> TLSIdentity? {
+        var osStatus: OSStatus
         let thisLabel : String? = "doco-sync-server"
-
+        
         //var thisData : CFData?
-    // tag::old-p2p-tlsid-tlsidentity-with-label[]
-    // tag::p2p-tlsid-check-keychain[]
+        // tag::old-p2p-tlsid-tlsidentity-with-label[]
+        // tag::p2p-tlsid-check-keychain[]
         // USE KEYCHAIN IDENTITY IF EXISTS
         // Check if Id exists in keychain. If so use that Id
-        do {
-            if let thisIdentity = try TLSIdentity.identity(withLabel: "doco-sync-server") {
-                print("An identity with label : doco-sync-server already exists in keychain")
-                return thisIdentity
-                }
-        } catch
-          {return nil}
+        if let tlsIdentity = try TLSIdentity.identity(withLabel: "doco-sync-server") {
+            print("An identity with label : doco-sync-server already exists in keychain")
+            return tlsIdentity
+        }
+        
         // end::p2p-tlsid-check-keychain[]
-        thisAuthenticator.ClientCertificateAuthenticator(identity: thisIdentity )
-        config.thisAuthenticator
-    // end::old-p2p-tlsid-tlsidentity-with-label[]
-
-
-// tag::p2p-tlsid-check-bundled[]
-// CREATE IDENTITY FROM BUNDLED RESOURCE IF FOUND
-
+        //        thisAuthenticator.ClientCertificateAuthenticator(identity: thisIdentity )
+        //        config.thisAuthenticator
+        //        FIXME: Not sure, whats done here? Is this client side / server side authenticator?
```

Comment:
@jayahariv This snippet is used here https://docs.couchbase.com/couchbase-lite/current/swift/p2psync-websocket-using-active.html#configuring-client-authentication in example 7